cdap-cli/src/main/java/co/cask/cdap/cli/ArgumentName.java

```diff
@@ -163,6 +163,9 @@ public String toString() {
     "'<namespace-name>.<artifact-name>.<artifact-version>' or " +
     "'<namespace-name>.<app-name>.<app-version>'.\n" +
     "\n" +
+    "Note:  Metadata for versioned entities such as applications, programs, schedules and program runs is not " +
```

Comment:
Suggest:

**Note**: Metadata for versioned entities is not versioned, including entities such as applications, programs, schedules, and program runs. Additions to metadata in one version are reflected in all versions.

Reply:
Done.